spec/enumerable_spec.rb

```diff
@@ -0,0 +1,794 @@
+# spec/enumerable_spec.rb
+require './main.rb'
```

Comment:
- [ ]  Using `require_relative` for files in your directory instead of `require` is a ruby convention.
- [ ] change that line to `require_relative '../main'`

Reply:
Change applied.